dCommon/GeneralUtils.h

```diff
@@ -18,6 +18,9 @@
 #include "dPlatforms.h"
 #include "Game.h"
 #include "Logger.h"
+#include "json.hpp"
```

Comment:
Should we really transitively include this everywhere?

Reply:
Hmmm, probably not, I could just switch it to the forward include

Reply:
that doesn't work :(

Reply:
potentially I want to use this for other things in other servers, so it would be needed in other places in the future

Reply:
I dont think we need to transitively include this everywhere yet.  When its needed it can be moved there, but for now let's put this only where necessary.

Reply:
though I will note this header is pre-compiled, so it doesn't entirely cost much to include it here to xiphos point, since its 3rdparty and not changing often

Reply:
cool deal, will do, may just make it part of the web API then until it's needed elsewhere